lianad/src/database/sqlite/mod.rs

```diff
@@ -481,6 +481,23 @@ impl SqliteConn {
         .expect("Database must be available")
     }
 
+    pub fn insert_outpoint_seen_before<'a>(
+        &mut self,
+        outpoints: impl IntoIterator<Item = &'a bitcoin::OutPoint>,
+    ) {
+        db_exec(&mut self.conn, |db_tx| {
+            for outpoint in outpoints {
+                db_tx.execute(
+                    "INSERT INTO payjoinoutpoints (txid, vout, added_at) \
+                        VALUES (?1, ?2, ?3)",
+                    rusqlite::params![outpoint.txid[..].to_vec(), outpoint.vout, curr_timestamp()],
```

Comment:
Something is not adding up to me. The schema has a single row for outpoint but here we are adding the two components seperatetly? There is probably some magic transformation somewhere?

Reply:
might be bc the old db tables still exist with the previous schema?